### PR TITLE
Adds command to modify the log level from the command shell

### PR DESCRIPTION
**Pull request by jhalexand**
This PR adds a command to the standard set of commands available in the command shell that allows you to modify the log level either for a specific target logger or for the log handlers. I've also added a stub of a unit test for the standard commands that currently only tests the log level command. If any standard commands are touched, we should add tests for them as we go.

### CODE REVIEW

File: olcut-core/src/main/java/com/oracle/labs/mlrg/olcut/command/CommandInterpreter.java [hunk 1]
```diff
@@ -1508,6 +1509,74 @@ public String unredir(CommandInterpreter ci) {
...
+                // or if the requested level is finer than the current level.
+                if (loggerName.isEmpty() || (logLevel.intValue() < currLevel.intValue())) {
+                    for (Handler h : handlers) {
+                        h.setLevel(logLevel);
+                    }
+                } else if (!loggerName.isEmpty() && logLevel.intValue() > currLevel.intValue()) {
+                    ci.out.println(logLevel.toString() + " is less fine than current "
+                            + currLevel.toString() + ", won't automatically deescalate the Handler level");
+                }
+            } catch (IllegalArgumentException e) {
```
Comment (Craigacp) [inline olcut-core/src/main/java/com/oracle/labs/mlrg/olcut/command/CommandInterpreter.java:1569]: Do we want the try to be this broad? Are there other things in here that could throw `IllegalArgumentException`?

File: olcut-core/src/test/java/com/oracle/labs/mlrg/olcut/command/StandardCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
+package com.oracle.labs.mlrg.olcut.command;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import jdk.jfr.internal.LogLevel;
```
Comment (Craigacp) [inline olcut-core/src/test/java/com/oracle/labs/mlrg/olcut/command/StandardCommandTest.java:6]: Did you mean to import JFR here?

**Review by Craigacp — CHANGES_REQUESTED** [review]
Couple of small things.

File: olcut-core/src/main/java/com/oracle/labs/mlrg/olcut/command/CommandInterpreter.java (hunk 1 above)
Comment (jhalexand) [reply olcut-core/src/main/java/com/oracle/labs/mlrg/olcut/command/CommandInterpreter.java]: I don't think anything else in there will throw that exception, but fair point anyway - I narrowed the scope of the try catch just to surround that parse.

File: olcut-core/src/test/java/com/oracle/labs/mlrg/olcut/command/StandardCommandTest.java (hunk 2 above)
Comment (jhalexand) [reply olcut-core/src/test/java/com/oracle/labs/mlrg/olcut/command/StandardCommandTest.java]: Good catch - seems like the IDE shouldn't have done that without telling me. It's gone now (it must have come in why I typed LogLevel after meaning to type LogRecord).

**Review by Craigacp — APPROVED** [review]
LGTM

**Pull request merged**